packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -222,6 +222,25 @@ For more information on the listed features, visit the [Okta Glossary](https://h
 
 This section helps you define how your customers get set up with your integration. Detail all settings and include any images that can assist the user. Include any best practices for your procedure, such as SCIM guidance on mappings for attributes, especially required attributes that don't have a default mapping.
 
+> **Note:** If you configure Okta as a Big Bang Service Provider, you need to provide a warning note to your customer. See [SAML configuration warning example](#saml-configuration-warning-example).
```

Comment:
In this case Okta is not Service provider, please update this sentence to:
**Note:** If your Service Provider configured as a "Big Bang", you need to provide a warning note to your customer. See [SAML configuration warning example](#saml-configuration-warning-example).

Reply:
@ykrestyanov-okta: Thank you, I'll update this. 
Does L229 make sense then? Can you please review?